src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-alter-books-table.yaml
```

Comment:
changeset file name should describe what changes were applied. The `02-alter-books-table` is too generic